rules-tests/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromMockObjectRector/Fixture/some_test_with_mock.php.inc

```diff
@@ -8,7 +8,7 @@ use PHPUnit\Framework\TestCase;
 
 final class SomeTestWithMock extends TestCase
 {
-    public function createMock()
+    public function test()
```

Comment:
I updated method name due to child without type is error early if parent method already have type, also consistent with example